src/app/dashboard/page.tsx

```diff
@@ -82,6 +82,7 @@ const Home: NextPage = () => {
     amt: parseFloat(dt.amount),
     name: new Date(dt.created).toISOString().slice(0, 10),
   }))
+  console.log('holderValue', holderValue)
```

Comment:
please delete this line

Reply:
<img width="865" alt="image" src="https://github.com/EveripediaNetwork/iq-ui/assets/67792966/6b9081a2-6acc-4686-9343-a23e4621cfbf">

@SidharthK2  if we are removing the day filter, why is this button for 1D still here? is it intentional?